### DISCUSSION TITLE
WS2814 FCOB

### DISCUSSION THREAD

**Discussion by TheOnlyOne93CJT** [awawa-dev/HyperHDR#938] [Q&A]
So I just got a strip of these WS2814 FCOBs. They're working fine in WLED with BRGW. In WLED you have to select the "swap Green and White" option, so I guess it becomes BRWG. There's no option for that in HyperHDR though.
Weirdly, in hyperion.ng it works perfectly fine with screen capture... I'm thinking maybe it's using the byte order that WLED set up. Is there any way for HyperHDR to just use whatever byte order WLED has set instead of setting its own?
I really dig the software but right now, white shows up as green and green shows up as white. This messes up all the other colors too, 'cause the byte order is expecting BRWG instead of BRGW.
Any ideas on how to fix this with these WS2814 FCOB strips?

**Comment by awawa-dev** [comment]
What is in the HyperHDR video live preview? Colors are OK? https://github.com/awawa-dev/HyperHDR/discussions/848

**Reply by TheOnlyOne93CJT** [reply]
colors look fine in the live preview. I ran into this issue last night at like 10 pm. At least honestly I think they looked fine. I will load it up again later today and double check. I did just hyperion.ng also and it worked perfectly fine. I assume it was because it doesn't have an option for byte order in it's led controller. So I'd guess it's just using exactly what WLED is using. I did notice that this specific feature to switch green and white got added in [WLED Beta Release 0.15.0-b1](https://github.com/Aircoookie/WLED/releases/tag/v0.15.0-b1)
ColoOrderMap W channel swap (color order overrides now have W swap)
But I can't be sure. I'm relatively new to setting up leds. But have plenty of experience with electronics in general. So if theres anything you need from me just ask. I'd love to get this working as the strips really look good. 

Edit : Forgot to mention I'm actually using the dx11 screen capture not usb capture. I also do not have HDR active on my computer. If that makes any difference.

**Comment by evelant** [comment]
I just got some of these strips too and would like to set them up with hyperhdr. @TheOnlyOne93CJT did you find a configuration that works?

**Reply by JeffMedd** [reply]
I'm also trying to get these to work- @TheOnlyOne93CJT  @evelant  did you manage to resolve this?

**Reply by evelant** [reply]
I haven't had time to play with it more, sorry, I still don't know if there's a working configuration for these strips

**Reply by JeffMedd** [reply]
@evelant @TheOnlyOne93CJT  I have now managed to get these working with HyperHDR. Have added a 'Swap W&G, W&B, W&R' option into the LED setup menu and tested it works. Let me know if you want more details and I can send it to you.
